crc_libvirt.template

```diff
@@ -93,12 +93,12 @@ cluster_create()
   </forward>
   <bridge name='tt0' stp='on' delay='0'/>
   <mac address='52:54:00:76:98:81'/>
-  <domain name='ReplaceMeWithCorrectVmName.tt.testing' localOnly='yes'/>
+  <domain name='ReplaceMeWithCorrectVmName.ReplaceMeWithCorrectBaseDomain/>
```

Comment:
why you removed `localOnly='yes'` this is important to have local dns communication.

Reply:
This got removed by mistake, I'll readd